### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by Aditya-Sakpal** [agama-project/agama#862] [General]
Hello Everyone , myself Aditya Sakpal while searching for a good repositories for making open source contributions I stumbled upon openSUSE's website and I found this repository . I'm good at react js and frontend development and I have made few open source contributions in the past and now I am excited make contributions here as well .

**Comment by dgdavid** [comment]
Hi @GIGA-CODER!

Thanks for showing interest in the project :love_you_gesture: Hopefully we are going to fill a few good-first-issues this week. Be sure to keep an eye out! :wink: 

We also welcome your feedback on open discussions and issues.

Regards!

**Comment by imobachgs** [comment]
Hi @GIGA-CODER,

Thanks for your interest in the project! May I suggest fixing issue #860? I expect the problem to be in the [L10n context](https://github.com/openSUSE/agama/blob/master/web/src/context/l10n.jsx). Please feel free to work on a different issue if you'd like. And, of course, do not hesitate to ask any questions you may have.

Thanks in advance!
